acra/configuring-maintaining/general-configuration/acra_connector.md

```diff
@@ -0,0 +1,337 @@
+---
+title: AcraConnector
+bookCollapseSection: true
+weight: 4
+---
+
+# AcraConnector
+
+## Command line flags
+
+* `--audit_log_enable`
+
+  Enable audit log functionality.
+  Default is `false`.
+
+* `--client_id=<id>`
+
+  Provided Client ID identifies Secure Session transport keys for handshake with AcraServer/AcraTranslator and will be sent to AcraServer/AcraTranslator as identifier of encryption keys
+
+* `--mode`
+
+  Expected mode of connection. 
+  * `AcraServer` - (default) mode switch AcraConnector to work with `AcraServer`. Only in this mode works next flags: `incoming_connection_port`/`incoming_connection_api_port`/`acraserver_connection_host`/`acraserver_connection_string`/`http_api_enable`/`acraserver_api_connection_string`/`acraserver_api_connection_port`/`acraserver_securesession_id`
+  * `AcraTranslator` - mode switch AcraConnector to work with `AcraTranslator`. Only in this mode works next flags: `acratranslator_connection_host`/`acratranslator_connection_string`/`acratranslator_connection_port`/`acratranslator_securesession_id`
```

Comment:
Here we can have a primitive table:

| AcraServer | AcraTranslator |
| --- | --- |
| incoming_connection_api_port | acratranslator_connection_host |
| acraserver_connection_host | acratranslator_connection_string |      
| acraserver_connection_string | acratranslator_connection_port |      
| http_api_enable | acratranslator_securesession_id |      

But I'm not sure if it will be rendered correctly for hugo

Reply:
what about moving "mode" flag to the end of section with table? and that is how it looks:
![image](https://user-images.githubusercontent.com/6237700/127941791-f1018681-bb2d-40e3-bd4f-bd0fef12a920.png)